server/data/nl/pcmill.txt

```diff
@@ -0,0 +1,1195 @@
+De lucht is helder; de sterren fonkelen.
+Wij zijn nog nooit in Azië geweest.
+Hoeveel groente eet jij per dag?
+Het geweld in sommige landen kennelijk een groot probleem.
+Landbouw was een grote factor in het ontstaan van de huidige wereld.
+De leerling was het niet eens met de leraar.
+Het jacht lag in de haven, het was wel twintig meter lang.
+De kermis was een week in het dorp.
+Ik wil je bedanken voor het bedrag dat je me gaf.
+De benzine is absoluut een stuk duurder geworden.
+Als je die specifieke arts wil zien moet je dat aanvragen.
+De vader moest het scherpe object afpakken van zijn zoontje.
+De typemachine werd eigenlijk nooit meer gebruikt.
+Er was voor een aanzienlijk bedrag geïnvesteerd in de handel.
+Als je over die heuvel loopt zie je het paleis vanzelf.
+De patiënt was zo ziek dat ze haar moesten optillen.
+Persoonlijk denk ik niet dat het veel gaat opleveren.
+De subsidie is redelijk moeilijk te krijgen.
+Het slachtoffer kreeg hulp van een psychiater.
+Tijdens Kerstmis zijn er gewoonlijk minder winkels open.
+De paashaas wordt steeds populairder tijdens Pasen.
+De dromen waren heel vreemd.
+Het meisje moest huilen, ze dacht dat haar vader weg was.
+De werkzaamheden zorgden voor een ontstopping in het verkeer.
+De voorstelling begint om acht uur.
+De offerte was gemaakt door een junior verkoper.
+De vacature was voor een medior developer.
+Eigenlijk hebben we voor deze uitdaging een senior nodig.
+Kan je de offertes doorsturen?
+Maak me wakker om zeven uur morgen.
+Zet een timer voor tien seconden.
+Hoeveel tijd is er nog resterend op mijn timer?
+Wat is de datum?
+Hoe laat is mijn wekker gezet?
+Verwijder mijn wekker.
+Wat is het weer vandaag?
+Wat is de weersverwachting deze week?
+Gaat het regenen maandag?
+Wat is het weer in Arnhem?
+Het tijdschrift wordt op bijna alle locaties verkocht.
+De doodskist was uitgebreid versiert.
+Door de dichte sluisdeuren kon het vrachtschip niet verder.
+Het binnenvaartschip lag vol met kolen.
+De processor heeft een kloksnelheid van twee komma acht gigahertz.
+Je kan best ver inzoomen op de kaart.
+Door uit te zoomen werd steeds meer van de afbeelding zichtbaar.
+Het kladblok stond vol met aantekeningen.
+Zet je ideeën maar op het whiteboard in de gang.
+Het systeemplafond was ingezakt door de waterschade.
+De schilder plakte het plafond af voordat hij verder ging met schilderen.
+De producer van de hit had al meer top tien muziek gemaakt.
+Die donuts zijn de grootste vijand van mijn dieet.
+De man typte op een mechanisch toetsenbord.
+De verpleegster hechtte de wond.
+De ruzie tussen de twee werd vervolgens uitgevochten op straat.
+Hoeveel wil je bieden voor dit schilderij?
+De felgekleurde caravan was de buurman een doorn in het oog.
+Het stoute jongetje werd door zijn oma gestraft.
+Het regelen van het afscheidsfeest kost veel tijd.
+De klinkers van de straat werden vervangen door asfalt.
+Wil je aspergesoep of tomatensoep?
+De toets werd goed gemaakt door de leerlingen.
```

Comment:
Is dit een Nederlandse uitdrukking? Ik zou eerder zeggen: 'Alle leerlingen scoorden goed op de toets.' In mijn ogen klinkt de 'goed gemaakt' hier als 'er wordt ijverig aan gewerkt.'